packages/api/src/external/carequality/organization-template.ts

```diff
@@ -187,5 +198,18 @@ function getEndpoint(oid: string, url: string, urlType: ChannelUrl) {
             </extension>
         </extension>
     </Endpoint>
-</contained>`;
+</contained>
+`;
+}
+
+function getPartOf(hostOrgOid?: string): string {
+  const partOfOid = hostOrgOid || metriportOid;
```

Comment:
nit: prefer `??`, as `||` is boolean based and `??` is specific for nullish values (which is the case here)

Also, why name it `partOfOid`? That's the full OID (`urn:oid:...` is the URN notation, with a namespace before the OID)

Reply:
bad naming, but the attribute in the xml is called `<partOf>`